### PR TITLE
adds white glubs to premium goods in autodrobe

### PR DESCRIPTION
**Pull request by ghost**
the PR which changed latex gloves from white to seethrough blue was the worst PR ever ever ever ever ever ever

:cl:
 * rscadd: adds white gloves to premium goods (insert coin) in autodrobe

### CODE REVIEW

**Comment by ghost** [comment]
#2196 THIS ONE
IT IS SATAN SPAWN

**Comment by Arthurdentist** [comment]
gonna need a server poll before you buff the autodrabe like this.

**Comment by NigglyWiggly** [comment]
No

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -2426,6 +2426,7 @@ var/global/num_vending_terminals = 1
 		/obj/item/clothing/suit/wizrobe/magician/fake = 3,
 		/obj/item/clothing/head/wizard/magician = 3,
 		/obj/item/clothing/suit/sakura_kimono = 3,
+		/obj/item/clothing/gloves/white = 3,
```
Comment (jknpj) [inline code/game/machinery/vending.dm:2429]: No need for this ,

Comment (ghost) [reply code/game/machinery/vending.dm:2429]: yeah no need for it but also no need not for it

Comment (jknpj) [reply code/game/machinery/vending.dm:2429]: /shrug

**Review by jknpj — APPROVED** [review]

**Pull request merged**